### PR TITLE
Fixes and improvements for Tenant Security page

### PR DESCRIPTION
**Pull request by Alevsk**
- tenant securityContext was only being applied to first pool
- fixed style issues on tenant security page to be more consistent
- added missing FsGroupChangePolicy in the SecurityContextSelector
  component

image

Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/securityContextSelector.tsx [hunk 1]
```diff
@@ -92,9 +101,33 @@ const SecurityContextSelector = ({
                 min="0"
               />
             </div>
+            <br />
+            <br />
+            <br />
+            <br />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/securityContextSelector.tsx:107]: Please use marginBottom for this

File: portal-ui/src/screens/Console/Tenants/types.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ export interface ISecurityContext {
   runAsGroup: string;
   runAsNonRoot: boolean;
   fsGroup: string;
-  fsGroupChangePolicy: "Always" | "OnRootMismatch";
+  fsGroupChangePolicy: string;
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/types.ts:140]: Please don't change this as it only accept those two values

File: portal-ui/src/screens/Console/Tenants/types.ts [hunk 3]
```diff
@@ -360,5 +360,6 @@ export interface IEditTenantSecurityContext {
   runAsUser: string;
   runAsGroup: string;
   fsGroup: string;
+  fsGroupChangePolicy: string;
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/types.ts:363]:
```suggestion
  fsGroupChangePolicy: "Always" | "OnRootMismatch";
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**